### DISCUSSION TITLE
如何调用妙言

### DISCUSSION THREAD

**Discussion by ghost** [tw93/MiaoYan#308] [Q&A]
如果我想在自己的博客上调用妙言的Markdown模板可否支持？或者是以插件的形式调用？

**Comment by tw93** [comment]
可以参考潮流周刊里面的展示，我用的是一个
https://github.com/tw93/weekly/tree/main/src/styles

或者你可以试试，通过妙言导出 html 可以看到里面的写法，这里很通用，类比于一个通用的markdown转html以后的渲染样式，规则基本上是统一的
